packages/debugger/lib/txlog/actions/index.js

```diff
@@ -211,6 +211,19 @@ export function logEvent(pointer, newPointer, step, decoding, rawEventInfo) {
   };
 }
 
+//this may be replaced once decoding info was added
+export const STORE = "TXLOG_STORE";
```

Comment:
Would you consider `TXLOG_STORAGE` or `TXLOG_STORAGE_SLOT` since it's bundling a storage slot and value?

Reply:
I like `RECORD_STORAGE`, but am ok if you think it's fine.

Reply:
I guess probably it should be `RECORD_STORE`, since it's recording a particular store.

Reply:
Yeah I think I'll just go ahead and merge this as-is. :P

Reply:
> I don't really understand the basis of your criticism.

Sorry, @haltman-at, I should have explained this better. It stems from this being a Redux action creator for `store`, which is how Redux refers to its state tree. However, I don't have a good alternate suggestion so, I can't complain.

Reply:
Oh wow -- yeah that's really far away from anything that was on my mind.  That's just on a totally different level of abstraction entirely.  I don't expect people to get confused by that collision because I don't expect people to be thinking about those simultaneously?